learn/wavemaker-release-notes/v11-4-3.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "WaveMaker 11.4.3 - Release date: 06 November 2023"
+id: "v11-4-3"
+sidebar_label: "v11.4.3"
+---
+
+WaveMaker has released version 11.4.3, which includes multiple bug fixes related to Studio Page Workspace, React Native, Cordova, and others.
+
+## Bug Fixes
+
+---
+
+- Fixed an issue related to Oracle Database by changing the Oracle java type from int to long when precision is 10. This fix is only applicable to the newly imported Oracle Database.
+- Fixed the temporary files clean-up issue after Excel and CSV file export.
+- Fixed Excel and CSV file export API to support exporting large files.
+
+### Studio Page Workspace
+
+- Fixed the issue in the Select Widget where once it is disabled, the Select Widget in Page Workspace is not selectable.
+- Fixed an issue in the Video widget where in the **Poster** property a prefix is getting added to the path.
+- Fixed an infinite loop issue that arises when a Partial in Page is called inside the same Partial.
+- Fixed the issue related to Prefab where the delete option was not visible to remove the Prefab.
+- Fixed the Chart-related issue where the values were getting displayed outside when the value was set to inside in Markup.
+- Fixed an issue where when navigated to Partial user was getting redirected to dialog in Partial but not the main content of the Partial.
+- Fixed an issue where the user was unable to view the fields from the columns tab.
+- Fixed an issue in Form and  Live Form where bindings on the Form field are getting empty after adding the Custom field and saving the advanced settings dialog.
+- Fixed the Rating Widget issue that was not working as expected.
+- Fixed an issue with viewing the Markup and script of a page.
+- Fixed Live filter issue where icon class is getting added by default.
+- Fixed an issue where the user is unable to navigate to the javascript function when the javascript event is clicked.
```

Comment:
Fixed an issue where on clicking on a Javascript event from events tab, the user cursor does not focus on the clicked javascript event.